core.lua

```diff
@@ -1,13 +1,27 @@
 local CoolLine = CreateFrame("Frame", "CoolLine", UIParent)
-
 CoolLine.MainFrame = CoolLine
 
 CoolLine:SetScript("OnEvent", function(this, event, ...)
 	this[event](this, ...)
 end)
 
-local IS_WOW_8 = GetBuildInfo():match("^8")
-local IS_WOW_CLASSIC = GetBuildInfo():match("^1")
+local current_version = tonumber(string.sub(GetBuildInfo(), 1, 1))
+local supported_retail_version = 9
+local supported_classic_version = 1
+local IS_RETAIL = current_version == supported_retail_version
+local IS_WOW_CLASSIC = current_version == supported_classic_version
+if not IS_RETAIL and not IS_WOW_CLASSIC then
+	print("CoolLine hasn't been updated to support WoW v"..GetBuildInfo())
+	print("Please file any bugs you find @ https://github.com/LoneWanderer-GH/CoolLine/issues")
```

Comment:
I realize you aren't playing any more, but I didn't want to assume you were giving up maintenance of the plugin.